tests/assets/layered_deployment.template_1.json

```diff
@@ -0,0 +1,51 @@
+{
+    "content": {
+        "modulesContent": {
+            "$edgeAgent": {
+                "properties.desired": {
+                    "modules": {
+                        "LayeredDeploymentTempSensor": {
+                            "settings": {
+                            "image": "mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:${LAYERED_RUNTIME_TAG}",
```

Comment:
do we own this image? Or do we plan on updating it for layered deployment? I think the simulated temperature sensor image only goes to 1.0 for now

Reply:
we do not. I copied the RUNTIME_TAG from the layered deployment tests and chose it to be separate for more control. but it is currently set to 1.0. I will change this to RUNTIME_TAG to keep consistency. this image is just for testing, so I can see the value in moving 1.0 to be included in this string but that's out of scope for this work.